shopinvader_wishlist/services/wishlist.py

```diff
@@ -265,25 +265,11 @@ def _json_parser_line(self):
             "id",
             "sequence",
             "quantity",
-            ("shopinvader_variant_id:product", self._json_parser_product()),
+            ("shopinvader_variant_id:product", self._json_parser_product),
```

Comment:
This line looks suspicious.... `_json_parser_product` is a method

Reply:
it's correct, that's a callable (supported in jsonify since some versions (see https://github.com/OCA/server-tools/pull/1829 :wink: ).
It will be supported in the refactoring by @len-foss as well.